scripts/qa_crowbarsetup.sh

```diff
@@ -2262,7 +2262,7 @@ function enable_ssl_generic()
             if [[ ! $(iscloudver 7plus) ]] ; then
                 return
             fi
-        ;;&
```

Comment:
@aplanas, You can find this described in more detail in the [Bash Reference Manual](https://www.gnu.org/software/bash/manual/bashref.html#Looping-Constructs).